src/workerd/api/BUILD.bazel

```diff
@@ -121,10 +129,48 @@ kj_test(
     ],
 )
 
+wd_cc_library(
+    name = "util",
+    srcs = ["util.c++"],
+    hdrs = ["util.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//src/workerd/jsg",
```

Comment:
Hope this isn't too much bikeshedding, just want to get this target right: @simdutf should be implementation_deps here since it's only used in the .c++ file

Reply:
The comments and the reviews are extremely helpful. Thank you! (And I don't consider this as bike-shedding)